apps/s_client.c

```diff
@@ -2013,10 +2022,14 @@ int s_client_main(int argc, char **argv)
                    "-dane_tlsa_domain option.\n", prog);
         goto end;
     }
+#ifndef OPENSSL_NO_DTLS
+    if (isdtls)
+        tfo = 0;
```

Comment:
Should this actually be an error?

Reply:
That's pretty subjective, but I suppose it could be.

Reply:
Fixed